security/cmd/chiron/main.go

```diff
@@ -50,7 +50,7 @@ var (
 	}
 
 	rootCmd = &cobra.Command{
-		Use:   "Istio Webhook Controller",
+		Use:   "chiron",
```

Comment:
Do we need to change this ? The rest of the code will go to operator.

Reply:
Done: removed this change.